src/ontology/reports/doid.subclass.added.robot.tsv

```diff
@@ -27,23 +27,18 @@ MONDO:0000297	baylisascariasis	MONDO:0005560	DOID:0050259	DOID:936	brain disorde
 MONDO:0000298	dioctophymiasis	MONDO:0005093	DOID:0050260	DOID:37	skin disorder
 MONDO:0000298	dioctophymiasis	MONDO:0005240	DOID:0050260	DOID:557	kidney disorder
 MONDO:0000301	ophthalmomyiasis	MONDO:0005328	DOID:0050268	DOID:5614	eye disorder
-MONDO:0000302	basidiobolomycosis	MONDO:0000255	DOID:0050278	DOID:0050135	subcutaneous mycosis
```

Comment:
This removal looks correct since MONDO:0000302 has the annotation excluded subClassOf subcutaneous mycosis